src/pages/Detail/DetailHotelRoom/DetailHotelRoom.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components';
+import { fontSet, BoxSizeSet, border } from '../../../styles/Mixins';
+
+export const DetailHotelRoom = () => {
+  return (
+    <RoomChoiceWrap>
+      <RoomChoiceTitle>객실선택</RoomChoiceTitle>
+      <RoomImage src="./images/hotel_1.jpg" alt="hotel" />
+      <RoomReservation color="#74B9ff">
+        날짜 선택하고 객실 더보기
+      </RoomReservation>
+      <ChoiceInfo>
+        캘린더에서 날짜를 선택하시고 특가 객실을 확인하세요
+      </ChoiceInfo>
+    </RoomChoiceWrap>
+  );
+};
+
+const RoomChoiceWrap = styled.div`
+  padding: 20px 30px;
+  border-top: 10px solid #f8f8f9;
+  border-bottom: 1px solid #e7e7e7;
+`;
+
+const RoomChoiceTitle = styled.h2`
+  ${fontSet('25px', 'inherit', '700')};
+  padding-top: 10px;
+`;
+
+const RoomImage = styled.img`
+  ${BoxSizeSet('100%', 'auto', '30px 0 0 0')}
+`;
+
+const RoomReservation = styled.button`
+  ${fontSet('inherit', ({ color }) => color, 'inherit', '0')};
```

Comment:
props로 넘겨준 것을 사용할 때는 `${(props)=> {}}` 이렇게 함수형태로 사용하면 됩니다!
위와 같은 경우에는 
```suggestion
  ${({ color }) => fontSet('inherit', color, 'inherit', '0')};
```
이런식으로 사용하면 되겠네요